docs/content/preview/integrations/cdc/debezium.md

```diff
@@ -123,6 +123,44 @@ curl -i -X POST -H "Accept:application/json" -H "Content-Type:application/json"
 }'
 ```
 
+{{< note title="Note" >}}
+
+In case you have a SSL enabled cluster, you need to provide the root certificate to the `database.sslrootcert` configuration property as well.
+
+{{< /note >}}
+
+**In case of SSL enabled YugabyteDB cluster, follow these steps:**
+
+* Copy the certificate file to your docker container (assuming that the file exists on the root directory of your machine)
```

Comment:
Docker is a product name, so we should capitalize it (unless they don't, but I think they do...)
```suggestion
* Copy the certificate file to your Docker container (assuming that the file exists on the root directory of your machine)
```

Reply:
Yes, they do capitalize it. Thank you for pointing out!